package.json

```diff
@@ -13,7 +13,9 @@
     "react-codemirror2@7.2.1": "patch:react-codemirror2@npm:7.2.1#./patches/react-codemirror2-npm-7.2.1-browserified.patch"
   },
   "scripts": {
-    "lint": "tsc --noemit && yarn eslint packages/*/src/**/*.{js,jsx,ts,tsx} && stylelint packages/*/src/**/*.css"
+    "lint": "tsc --noemit && yarn eslint 'packages/*/src/**/*.{js,jsx,ts,tsx}' && stylelint 'packages/*/src/**/*.css'",
```

Comment:
idk but it seems faster to let eslint and stylelint to resolve the glob pattern than by unpacking it via shell